src/main/webapp/js/user-page-loader.js

```diff
@@ -32,15 +32,15 @@ function setPageTitle() {
 /**
  * Shows the message form if the user is logged in and viewing their own page.
```

Comment:
According to step 5, this function shows the form as long as the user is logged in, even if they aren't viewing their own page. Please remove the part of the comment specifying that they have to view their own page.
https://sites.google.com/codeustudents.com/spring-2019/weeks-1-2-familiarization-projects/direct-messages#h.p_VLKzeyLFYYun